builder/ebssurrogate/builder.go

```diff
@@ -420,34 +466,15 @@ func (b *Builder) Run(ctx context.Context, ui packersdk.Ui, hook packersdk.Hook)
 			EnableAMISriovNetSupport: b.config.AMISriovNetSupport,
 			EnableAMIENASupport:      b.config.AMIENASupport,
 		},
-		&StepSnapshotVolumes{
-			PollingConfig:   b.config.PollingConfig,
-			LaunchDevices:   launchDevices,
-			SnapshotOmitMap: b.config.LaunchMappings.GetOmissions(),
-			SnapshotTags:    b.config.SnapshotTags,
-			Ctx:             b.config.ctx,
-		},
+		volumeMethod,
 		&awscommon.StepDeregisterAMI{
 			AccessConfig:        &b.config.AccessConfig,
 			ForceDeregister:     b.config.AMIForceDeregister,
 			ForceDeleteSnapshot: b.config.AMIForceDeleteSnapshot,
 			AMIName:             b.config.AMIName,
 			Regions:             b.config.AMIRegions,
 		},
-		&StepRegisterAMI{
```

Comment:
I would say yes since the create image also register the image for you.  See below
 
"Note
For Amazon EBS-backed instances, CreateImage creates and registers the AMI in a single request, so you don't have to register the AMI yourself. We recommend that you always use CreateImage unless you have a specific reason to use RegisterImage."

https://docs.aws.amazon.com/cli/latest/reference/ec2/register-image.html#:~:text=For%20Amazon%20EBS%2Dbacked%20instances,an%20AMI%20at%20any%20time.

Reply:
As confirmation, you can also see that the Deregister step is part of the `ebs` builder, which also uses CreateImage.

https://github.com/hashicorp/packer-plugin-amazon/blob/7710fbc3554e464faca17de05dd9e820e062a829/builder/ebs/builder.go#L409-L415